airflow/www/views.py

```diff
@@ -3026,6 +3026,14 @@ def tree_data(self):
         # avoid spaces to reduce payload size
         return htmlsafe_json_dumps(tree_data, separators=(',', ':'))
 
+    def warn_deployment_exposure(self):
+        deployment_warning = "Deployment is exposed to public internet"
+        flash(
```

Comment:
Flask uses cookie-based sessions. This warning will therefore only be displayed to clients who have visited `/robots.txt`, in our case bot crawlers. I think we should notify the administrator about this situation, not the bot.

Reply:
what would be the best way to display the message for admin team? is there some way to post the message only for admin sessions?

Reply:
I guess you need to write it down to the database as a new table or to the event log table.